src/optimizer/query_rewrite.c

```diff
@@ -6451,6 +6461,14 @@ qo_do_auto_parameterize (PARSER_CONTEXT * parser, PT_NODE * where)
 	      continue;
 	    }
 
+	  if (PT_EXPR_INFO_IS_FLAGED (dnf_node, PT_EXPR_INFO_DONT_AUTOPARAM))
+	    {
+	      /* copy_pull term from select list of derived table do NOT auto_parameterize */
+	      /* because the query rewrite step is performed in the XASL generation of DELETE and UPDATE. */
+	      /* to_do: remove rewriting aptr in the XASL generation of DEL,UPD (pt_to_delete_xasl) */
```

Comment:
In the future, we need to modify the front of  the part of causing error to reduce side effect for several cases we didn't find yet.
But this work will need more time to find a way to modify.

Reply:
Yes. I will...